### PR TITLE
catchup Embulk 0.10.29

### PR DESCRIPTION
**Pull request by hieudion**
Catch up this connector with embulk new version

### CODE REVIEW

File: build.gradle
Comment (vietnguyen-td) [inline build.gradle]: please use 0.3.0

File: gradle/wrapper/gradle-wrapper.properties
Comment (vietnguyen-td) [inline gradle/wrapper/gradle-wrapper.properties]: could you change to gradle 6++?

**Review by vietnguyen-td — COMMENTED** [review]
general LGTM, just minor comment about version

**Comment by hieudion** [comment]
@ylzen
Could you help to take a look when you have time?

**Review by vietnguyen-td — APPROVED** [review]
LGTM

**Review by vietnguyen-td — APPROVED** [review]
LGTM

**Review by vietnguyen-td — APPROVED** [review]
LGTM

File: .github/workflows/build.yml
Comment (dmikurube) [inline .github/workflows/build.yml]: Can you add a Windows environment to test as it had AppVeyor CI for Windows? It can be done with `matrix` like : https://github.com/embulk/embulk-standards/blob/v0.10.36/.github/workflows/check.yml#L11-L15

File: build.gradle
Comment (dmikurube) [inline build.gradle]: `jcenter()` can be removed.

File: build.yml
Comment (dmikurube) [inline build.yml]: Is this file on the root dir needed?

File: src/test/java/org/embulk/output/azure_blob_storage/TestAzureBlobStorageFileOutputPlugin.java
Comment (dmikurube) [inline src/test/java/org/embulk/output/azure_blob_storage/TestAzureBlobStorageFileOutputPlugin.java]: Duplicated?

**Review by dmikurube — APPROVED** [review]
Basically looks okay, so approved to unblock, but added some comments. Please address them before merging.

File: build.gradle
Comment (dmikurube) [reply build.gradle]: Done by myself in f17c4424763064b07513e990b879279efcdc3108

File: build.gradle
Comment (dmikurube) [inline build.gradle:135]: Sorry if I have already asked for this before about another plugin. Is it okay to include your name and address (from Git log) like this, in `pom.xml` in Maven Central?

Comment (dmikurube) [inline build.gradle:135]: @xuantuan58 Sorry if I have already asked for this before about another plugin. Is it okay to include your name and address (from Git log) like this, in `pom.xml` in Maven Central?

**Comment by dmikurube** [comment]
You might have a direct push to `master`, and that is causing the conflict?

**Comment by hieudion** [comment]
Ignore the Window test and will fix it later.

**Comment by dmikurube** [comment]
Approved to go with ignoring the Windows testing later.

**Pull request merged**
